pkg/csi/service/vanilla/controller.go

```diff
@@ -1103,7 +1568,16 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 			return c.createFileVolume(ctx, req)
 		}
 		volumeType = prometheus.PrometheusBlockVolumeType
-		return c.createBlockVolume(ctx, req)
+		if multivCenterCSITopologyEnabled {
+			if len(c.managers.VcenterConfigs) > 1 {
+				return c.createBlockVolumeWithPlacementEngineForMultiVC(ctx, req)
+			} else {
+				return nil, csifault.CSIUnimplementedFault, logger.LogNewErrorCode(log, codes.Unimplemented,
+					"CreateVolume in a single VC environment when multi-VC FSS enabled is not yet implemented")
```

Comment:
@shalini-b If in future we enable FSS then in case of multi VC deployments, we will end up getting this error during create volume? For a single VC deployment we must always have the FSS disabled as per line 1576.